server/@types/manage-recalls-api/models/MissingDocumentsRecord.ts

```diff
@@ -4,7 +4,7 @@
 
 export type MissingDocumentsRecord = {
     missingDocumentsRecordId: string;
-    categories: Array<'CHARGE_SHEET' | 'CPS_PAPERS' | 'DOSSIER' | 'DOSSIER_EMAIL' | 'EXCLUSION_ZONE_MAP' | 'LETTER_TO_PRISON' | 'LICENCE' | 'MISSING_DOCUMENTS_EMAIL' | 'NSY_REMOVE_WARRANT_EMAIL' | 'OASYS_RISK_ASSESSMENT' | 'PART_A_RECALL_REPORT' | 'POLICE_REPORT' | 'PREVIOUS_CONVICTIONS_SHEET' | 'PRE_SENTENCING_REPORT' | 'REASONS_FOR_RECALL' | 'RECALL_NOTIFICATION' | 'RECALL_NOTIFICATION_EMAIL' | 'RECALL_REQUEST_EMAIL' | 'REVOCATION_ORDER' | 'RESCIND_REQUEST_EMAIL' | 'RESCIND_DECISION_EMAIL' | 'OTHER' | 'UNCATEGORISED'>;
+    categories: Array<'CHARGE_SHEET' | 'CPS_PAPERS' | 'DOSSIER' | 'DOSSIER_EMAIL' | 'EXCLUSION_ZONE_MAP' | 'LETTER_TO_PRISON' | 'LICENCE' | 'MISSING_DOCUMENTS_EMAIL' | 'NSY_REMOVE_WARRANT_EMAIL' | 'OASYS_RISK_ASSESSMENT' | 'PART_A_RECALL_REPORT' | 'POLICE_REPORT' | 'PREVIOUS_CONVICTIONS_SHEET' | 'PRE_SENTENCING_REPORT' | 'REASONS_FOR_RECALL' | 'RECALL_NOTIFICATION' | 'RECALL_NOTIFICATION_EMAIL' | 'RECALL_REQUEST_EMAIL' | 'REVOCATION_ORDER' | 'RESCIND_REQUEST_EMAIL' | 'RESCIND_DECISION_EMAIL' | 'NOTE_DOCUMENT' | 'OTHER' | 'UNCATEGORISED'>;
```

Comment:
Not breaking anything, but this doc type will never be sent with a missing document record

Reply:
👍  though with swagger-to-ts I'll need to also adjust the API somehow to exclude from this list when regenerated?

Reply:
Keeping for now: the build fails without this included and for we seem to have all server side categories included here, including generated documents: subsetting seems not limited to Notes feature?
("server/controllers/documents/download/helpers/index.ts:58:97 - error TS2345: Argument of type 'category' is not assignable to parameter of type ")